### DISCUSSION TITLE
Will there be more basic beginner tutorials in the future?

### DISCUSSION THREAD

**Discussion by vinit-AM** [leap71/PicoGK#16] [General]
Beginners to coding, one might feel difficulty to understand how different object deal/use/code can together. Might documentation can help regarding same!

**Comment by josefinelissner** [comment]
Hi, @VinitCAd !
We will be releasing more tutorials and code examples in the future. Many people have asked for a very basic tutorial, so this is on the list for the next month or so.

**Reply by lasainr2023** [reply]
True, eager to see if I can apply PicoGK to my use case too

**Comment by Hnf95** [comment]
Yes I am kind of new to coding and I am a bit lost. Really need the tutorials.

**Reply by josefinelissner** [reply]
Hi, @Hnf95 !
If you are completely new to coding, I recommend going through some foundational tutorials in order to learn how to use / start a project in Visual Studio and how to write basic C# code. These are things that you are expected to be familiar with before diving into PicoGK and Computational Engineering.

Maybe this provides a starting point:
- https://www.w3schools.com/cs/index.php

**Comment by LinKayser** [comment]
Maybe this is what you are looking for, maybe it is currently too basic for what you are after. 

But I have just committed to writing a book, releasing one chapter per week (wish me luck), on this topic. I will start simple, but will progress a comprehensive introduction for Computational Engineering projects with PicoGK. 

[Here is the thread in the forum.](https://github.com/leap71/PicoGK/discussions/24)

Enjoy,
Lin
